modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/FileEntryContentDashboardItem.java

```diff
@@ -297,8 +302,12 @@ public String getScopeName(Locale locale) {
 	public Map<String, Object> getSpecificInformation(Locale locale) {
 		return HashMapBuilder.<String, Object>put(
 			"extension", _getExtension()
+		).put(
```

Comment:
@cgoncas I wonder if we can reorder this object to match the UI.
If not, I could force it in the FE but IDK if it makes a lot of sense

Reply:
Yeahh!! I will sort it in the backend.